FFXIVClientStructs/FFXIV/Client/UI/Misc/RaptureHotbarModule.HotBarSlot.cs

```diff
@@ -0,0 +1,307 @@
+using FFXIVClientStructs.FFXIV.Client.Game;
+using FFXIVClientStructs.FFXIV.Client.System.Framework;
+using FFXIVClientStructs.FFXIV.Client.System.String;
+
+namespace FFXIVClientStructs.FFXIV.Client.UI.Misc; 
+
+// ToDo: Wrap in RaptureHotbarModule partial struct for namespacing (API 10)
+[StructLayout(LayoutKind.Explicit, Size = Size)]
+public unsafe partial struct HotBarSlot {
+    public const int Size = 0xE0;
+
+    /// The string that appears when a hotbar slot is hovered over.
+    ///
+    /// Calculated by concatenating GetDisplayNameForSlot with PopUpKeybindHint (in most cases).
+    [FieldOffset(0x00)] public Utf8String PopUpHelp; // TODO (apiX): Rename to DisplayName
```

Comment:
Why not make a new field and mark the old one obsolete?
```suggestion
    [FieldOffset(0x00), Obsolete("Use DisplayName")] public Utf8String PopUpHelp;
    /// <inheritdoc cref="PopUpHelp"/>
    [FieldOffset(0x00)] public Utf8String DisplayName;
```

Reply:
I'm still not certain if renaming the field is the right move. This field controls this specific value:

![image](https://github.com/aers/FFXIVClientStructs/assets/5192145/639ad3d5-ff24-44f6-98c6-9bd7bd6a9a60)

(Also, this is not a net-new field, that existed before this PR. I just moved the file.)

Reply:
Does this include the hotkey hint? Could probably be named `TooltipText` instead.

Reply:
Might be confused with the *actual* tooltip, which doesn't actually use that name at all:

![image](https://github.com/aers/FFXIVClientStructs/assets/5192145/5bb81ecb-72a3-412a-83db-ae225e8f13e0)

Naming things hard, and I don't want to deal with it right now because it isn't net-new.

Reply:
Both are in theory a Tooltip, but I would probably keep the name as is since the other one is more of a full Tooltip than just the Action name and hotkey

Reply:
Removed pending rename due to maintainer request.

Reply:
One is AtkTooltip, the other is ItemDetail. 🤓☝️